march_joint_inertia_controller/CMakeLists.txt

```
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(march_joint_inertia_controller)
+
+add_compile_options(-std=c++14 -Wall -Wextra -Werror)
+
+find_package(catkin REQUIRED COMPONENTS
+  controller_interface
+  pluginlib
+  roscpp
+)
+
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES ${PROJECT_NAME}
+)
+
+include_directories(include SYSTEM ${catkin_INCLUDE_DIRS})
+
+add_library(joint_inertia_controller_lib src/joint_inertia_controller.cpp)
+target_link_libraries(joint_inertia_controller_lib ${catkin_LIBRARIES})
```

I'm sorry, I told you in my previous comment to change line 14 to `${PROJECT_NAME}`, but that also means changing these lines :see_no_evil: Next time we should try building before doing these kind of changes. I think this is the last issue :crossed_fingers: 

```suggestion
add_library(${PROJECT_NAME} src/joint_inertia_controller.cpp)
target_link_libraries(${PROJECT_NAME} ${catkin_LIBRARIES})
```